openedx/core/djangoapps/content_tagging/types.py

```diff
@@ -3,6 +3,7 @@
 """
 from typing import Union
 
-from opaque_keys.edx.keys import LearningContextKey, UsageKey
+from opaque_keys.edx.keys import CourseKey, UsageKey
+from opaque_keys.edx.locator import LibraryLocatorV2
 
-ContentKey = Union[LearningContextKey, UsageKey]
+ContentKey = Union[LibraryLocatorV2, CourseKey, UsageKey]
```

Comment:
What's wrong with leaving `[LearningContextKey, UsageKey]` ? Less specific?

Reply:
We need an Org to validate the permissions.
The org property is defined in the `CourseKey` and the `LibraryLocatorV2` for our cases. For the `UsageKey`, we get the org from the `context_key` (and we also check if the `contet_key` is a `CourseKey` or `LibraryLocatorV2`).

We could use `LearningContext` and duck typing to check if we have the org field (directly in the key or in its contex_key).

I personally prefer strict typing to make sure the use conforms with what we are implementing and testing here.